app/models/document.rb

```diff
@@ -209,12 +226,11 @@ class NotArchiveableError < StandardError; end
   validate :numbered
   validate :created_date_is_in_the_past
 
-  default_scope -> { no_owner.or(not_excluded_owners) }
-
```

Comment:
This needed to be removed so the uploads engine could find all documents for checking the blob key